component/sections/pricing/variant_c.js

```diff
@@ -87,14 +90,24 @@ function VariantC({ caption, title, description, plans }) {
                       {plans?.[0]?.primaryButton?.label && (
                         <a
                           className="block mt-6 w-full py-2 px-6 rounded-l-xl rounded-t-xl bg-webriq-blue hover:bg-webriq-darkblue text-white font-bold leading-loose transition duration-200"
-                          target={plans?.[0]?.primaryButton?.pageAccess === "openLinkToNewTab" ? "_blank" : null}
-                          rel={plans?.[0]?.primaryButton?.pageAccess === "openLinkToNewTab" ? "noopener noreferrer" : null}
+                          target={
```

Comment:
This **_target_** attribute value needs to be updated. Need to change target value to 
`plans?.[0]?.primaryButton?.linkTarget`.